packages/genesys-spark-components/src/components/stable/gux-form-field/functional-components/gux-form-field-visual-label/gux-form-field-visual-label.scss

```diff
@@ -2,33 +2,34 @@
 
 /* stylelint-disable-next-line selector-class-pattern */
 @mixin Style {
-  .gux-form-field-legend-label {
-    ::slotted(label) {
-      white-space: nowrap;
-    }
-
-    &.gux-required {
-      &::after {
-        color: var(--gse-ui-formControl-label-indicator-requiredColor);
-        content: ' *';
-      }
-    }
+  .gux-form-field-visual-label {
+    display: inline-flex;
+    flex: 1 0 auto;
+    gap: var(--gse-ui-formControl-label-gap);
 
     &.gux-beside {
       position: relative;
       top: var(--gse-ui-formControl-input-top);
       float: left;
       width: fit-content;
-      min-width: 45px; // TODO:2289
-      margin-right: var(--gse-ui-formControl-label-gap);
+      min-width: var(--gse-ui-formControl-input-textfield-minWidth);
+      max-width: fit-content;
+      margin-right: var(--gse-ui-formControl-group-gapItems);
     }
 
     &.gux-above {
       padding: var(--gse-ui-formControl-label-padding);
     }
 
     &.gux-screenreader {
-      @include mixins.gux-sr-only;
+      display: none;
+    }
```

Comment:
I don't know whether it makes more sense to hide the visual label here for screenreader only or with a function on each component. Any thoughts welcome